buildbuddy.yaml

```diff
@@ -60,6 +60,29 @@ actions:
         -//enterprise/server/remote_execution/commandutil:all
         -//enterprise/server/remote_execution/runner:all
         -//enterprise/server/test/integration/...
+  - name: Test (darwin_arm64)
+    os: "darwin"
+    arch: "arm64"
+    triggers:
+      push:
+        branches:
+          - "master"
+      pull_request:
+        branches:
+          - "*"
+    # TODO: Fix the tests below on Mac, and re-enable.
+    bazel_commands:
+      - >-
+        test --config=mac-workflows --test_tag_filters=-performance,-webdriver,-docker,-bare
+        --
+        //...
+        -//server/backends/disk_cache:all
```

Comment:
Could we use a tag to exclude these so we don't need to update this workflow file or does that not work?

Reply:
I think that should work - will send a separate PR for that